packages/good-design/src/advanced/customswitch/CustomSwitch.tsx

```diff
@@ -1,92 +1,92 @@
-import React, { useState, useCallback } from 'react'
-import { Pressable, Icon, View, Box } from 'native-base'
-import { first } from 'lodash'
-import SelectBox from './SelectBox'
+import React, { useState, useCallback } from "react";
+import { Pressable, Icon, View, Box } from "native-base";
+import { first } from "lodash";
+import SelectBox from "./SelectBox";
 // import SwitchIcon from ' ../../assets/svg/arrow-swap.svg'
 
 const SwitchIcon = () => (
   <g strokeLinecap="round" strokeLinejoin="round" strokeWidth="1.5">
-    <path d="m5 10.0909h14l-5.8396-5.0909" stroke="#B0B4BB"/>
-    <path d="m19 13.9091h-14l5.8396 5.0909" stroke="#B0B4BB"/>
+    <path d="m5 10.0909h14l-5.8396-5.0909" stroke="#B0B4BB" />
+    <path d="m19 13.9091h-14l5.8396 5.0909" stroke="#B0B4BB" />
   </g>
-)
+);
 
-const SelectListItem = (
-  {
-    chain,
-    onPress,
-    isListItem,
-    isListOpen,
-    isLeft,
-  }: {
-    chain: string | undefined,
-    onPress: ((isLeft: boolean) => void) | ((isLeft: boolean, chain: string) => void)
-    isListItem: boolean,
-    isListOpen: boolean,
-    isLeft: boolean,    
-  }) => {
-  const type = isListItem ? "list" : "button"
-  const onItemPress = useCallback(() => onPress(isLeft, chain ?? ''), [chain, onPress]);
+const SelectListItem = ({
+  chain,
+  onPress,
+  isListItem,
+  isListOpen,
+  isLeft
+}: {
+  chain: string | undefined;
+  onPress: ((isLeft: boolean) => void) | ((isLeft: boolean, chain: string) => void);
+  isListItem: boolean;
+  isListOpen: boolean;
+  isLeft: boolean;
+}) => {
+  const type = isListItem ? "list" : "button";
```

Comment:
the code is too "compressed" in this file 

please add newlines between all code blocks

Reply:
done